opentelemetry-journald-logs/examples/basic-logs.rs

```diff
@@ -0,0 +1,29 @@
+// cargo run --example basic-logs
+
+use opentelemetry_appender_tracing::layer;
+use opentelemetry_journald_logs::JournaldLogExporter;
+use opentelemetry_sdk::logs::LoggerProvider;
+use tracing::info;
+use tracing_subscriber::prelude::*;
+
+fn init_logger() -> LoggerProvider {
+    let exporter = JournaldLogExporter::builder()
+        .identifier("opentelemetry-journal-exporter")
```

Comment:
Assign `with` prefix to the builder methods?

Reply:
Done